test.php

```
@@ -0,0 +1,9 @@
+<?php declare(strict_types = 1);
+
+class HelloWorld
+{
+    public function sayHello(DateTimeImutable $date): void
```

PHPStan (2.1.17)

Parameter $date of method HelloWorld::sayHello() has invalid type DateTimeImutable.


(class.notFound)